docs/visualization/plotting-nd-data.ipynb

```diff
@@ -68,9 +66,26 @@
     "\n",
     "- Each dimension comes with two sliders to control the position of the slice and its thickness.\n",
     "\n",
-    "- Upon figure creation, the thickness is set to cover the entire range of the additional dimension, and the data is summed along these dimensions.\n",
+    "- Upon figure creation, the thickness is set to zero for the additional dimension.\n",
```

Comment:
This is not striclty zero, it is selecting the first bin. So the thickness is whatever the width of that bin is (as reflected in the range of values displayed next to the slider).
Then, when we increase the thickness, it is selecting integer numbers of bins.